site/src/components/PageHeader/PageHeader.tsx

```diff
@@ -60,5 +65,11 @@ const useStyles = makeStyles((theme) => ({
 
   actions: {
     marginLeft: "auto",
+
+    [theme.breakpoints.down("md")]: {
+      marginTop: theme.spacing(3),
```

Comment:
This extra margin is now pushing our primary content down when `actions` are not defined (this is visible in the updated stories). Maybe we should conditionally render the `Stack` containing the `actions`.

Reply:
Good catch!